src/main/java/bio/terra/cli/businessobject/resource/BqResolvedOptions.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.cli.businessobject.resource;
+
+/** This enum specifies the possible ways to resolve a BigQuery resource. */
+public enum BqResolvedOptions {
+  FULL_PATH, // For data table: [project id].[dataset id].[data table id if applicable]
+  TABLE_ID_ONLY, // [data table id]
+  DATASET_ID_ONLY, // [dataset id]
+  PROJECT_ID_ONLY; // [project id]
```

Comment:
I missed the earlier PR so I might not have context but I'm a little surprised here - how do you resolve a BQ resource from just a project ID?

Reply:
Maybe the command name is a little misleading. `terra resource resolve` returns the cloud identifier for a particular workspace resource. For different resources, we have flags that can tweak the behavior (e.g. `--exclude-bucket-prefix` drops the `gs://` from the bucket url and just gives the bucket name). The solutions team mentioned that sometimes they need the BQ project/dataset id separately, and so we added an option to return just parts of the BQ `FULL_PATH` to make those use cases easier. Before they were getting `[project id].[dataset id]` and parsing it into pieces in order to use it in some places.